src/libraries/System.Private.CoreLib/src/System/Runtime/Intrinsics/Arm/AdvSimd.cs

```diff
@@ -7442,6 +7442,230 @@ public new abstract class Arm64 : ArmBase.Arm64
         /// </summary>
         public static Vector128<float> ReciprocalStep(Vector128<float> left, Vector128<float> right) => ReciprocalStep(left, right);
 
+        /// <summary>
+        /// uint8x8_t vsli_n_u8(uint8x8_t a, uint8x8_t b, __builtin_constant_p(n))
```

Comment:
@TamarChristinaArm Everything looks good.

Just one question here - what is `__builtin_constant_p(n)` and
should we use `uint8x8_t vsri_n_u8 (uint8x8_t a, uint8x8_t b, const int n)` instead as we did for the remaining intrinsics with an immediate operand?

Reply:
> Just one question here - what is `__builtin_constant_p(n)` and

It's a compiler builtin supported by `clang` and `gcc` to indicate that the value of `n` must be a compile time constant. the problem is that in C there's no real way to indicate this as `const` only means the value is read-only in the function.

> should we use `uint8x8_t vsri_n_u8 (uint8x8_t a, uint8x8_t b, const int n)` comment here instead as we did for the remaining intrinsics with an immediate operand?

probably yes since in CLR it doesn't have to be since you generate that lookup table.  I generate these signatures through a script that reads the ACLE sources so they got in automatically :)